tests/switchless_worksleep/enc/enc.cpp

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#include <openenclave/corelibc/string.h>
+#include <openenclave/enclave.h>
+#include <openenclave/internal/print.h>
+#include "switchless_worksleep_t.h"
+
+void enc_ecall1_switchless(void)
+{
+    host_ocall1_switchless();
```

Comment:
What is the reason for doing an ocall here? We know the host thread is awake here because it just make the ecall. I don't understand what is being tested.

Reply:
The purpose of doing ocall here is to want the tasks to include the switchless ocall and the switchless ocall.